dev/ci/gitlab.bat

```diff
@@ -54,9 +54,11 @@ IF "%WINDOWS%" == "enabled_all_addons" (
     -addon=flocq ^
     -addon=interval ^
     -addon=gappa_tool ^
-    -addon=gappa
+    -addon=gappa ^
+    -addon=elpi ^
+    -addon=HB
 ) ELSE (
-  SET "EXTRA_ADDONS= "
+  SET "EXTRA_ADDONS= -addon=elpi -addon=HB"

```

Comment:
I think it would require some justification that these plugins are the only ones which are always build. Also if this is intended, they should got directly to the command line of MakeCoq_MinGW.bat (there used to be a few, e.g. ltac2).

Reply:
This is part of the last commit, that is meant to be removed before merging.
In this way I get CI produce the installer for me and I can download and test the artifact.

I see no reason to keep elpi and HB being built by default

Reply:
I remove this